lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -172,6 +184,19 @@ private function import($filepath, $sourceName, $indexDuringImport = false)
     $import->searchIndexingDisabled = !$indexDuringImport;
 
     $import->csv($fh);
+
+    // Update actor relationships in Elasticsearch
+    if ($indexDuringImport)
+    {
+      $this->log('Updating Elasticsearch actor relation data...');
+
+      foreach ($import->status['actorIds'] as $actorId)
+      {
+        $actor = QubitActor::getById($actorId);
+        arUpdateEsActorRelationsJob::updateActorRelationships($actor);
+        Qubit::clearClassCaches();
+      }
+    }
```

Comment:
Hi @mcantelon, why is this not performed from QubitActor's `save()` method when the actors are created?

Reply:
Hi Radda... With the authority record relation import we're just importing relations between existing actors and not modifying the actors themselves. We could though, in the above code, simply re-save the actors instead of explicitly calling ` arUpdateEsActorRelationsJob::updateActorRelationships` to get the same effect.

Reply:
Oh! I see, actually what I mentioned in https://github.com/artefactual/atom/pull/1074#pullrequestreview-408761222 and it looks like it wasn't a good idea to disregard it, sorry about that.

Reply:
No worries... I'd meant to love at that too and ended up forgetting about it too!